DESCRIPTION

```diff
@@ -107,6 +108,7 @@ Collate:
     'plot.lpeer.R'
     'plot.peer.R'
     'plot.pfr.R'
+    'poridge.R'
     'postprocess.pfr.R'
     'predict.fgam.R'
     'predict.pfr.R'
```

Comment:
This probably needs 

`Suggests: dtw`

as well, since the package is required for the examples